pkg/config/provider.go

```diff
@@ -52,9 +52,9 @@ type Provider struct {
 	// for "aws_rds_cluster", we drop "aws_" prefix and its group ("rds") to set
 	// Kind of the resource as "Cluster".
 	TerraformResourcePrefix string
-	// GroupSuffix is the suffix to append to resource groups, e.g.
-	// ".aws.tf.crossplane.io". Defaults to ".<prefix>.tf.crossplane.io".
-	GroupSuffix string
+	// RootGroup is the suffix to append to resource groups, e.g.
+	// "aws.tf.crossplane.io". Defaults to "<prefix>.tf.crossplane.io".
```

Comment:
Here, I guess we are referring to the `prefix` argument passed to `config.NewProvider` with `<prefix>`.  We may make this clear.